src/Shared/HttpSys/RequestProcessing/HeaderEncoding.cs

```diff
@@ -14,16 +15,7 @@ internal static class HeaderEncoding
 
         internal static unsafe string GetString(byte* pBytes, int byteCount)
```

Comment:
Do we even need these helpers?

Reply:
They're used in several places, I'll keep them.